litho-processor/src/main/java/com/facebook/litho/specmodels/model/PropDefaultModel.java

```diff
@@ -53,6 +96,8 @@ public int hashCode() {
     int result = mType.hashCode();
     result = 17 * result + mName.hashCode();
     result = 31 * result + mModifiers.hashCode();
+    result = 43 * result + mResType.hashCode();
+    result = 65 * result + mResId;
```

Comment:
nitpick - for consistency, can you replace 65 with 47 or 53 or 59 or 61 to keep it as prime numbers?